docs/src/description/keywords.md

```diff
@@ -11,8 +11,10 @@ AirScript defines the following keywords:
 - `integrity_constraints`: used to declare the [source section](./structure.md#source-sections) where the [integrity constraints are described](./constraints.md#integrity_constraints).
 - `let`: used to declare intermediate variables in the boundary_constraints or integrity_constraints source sections.
 - `periodic_columns`: used to declare the [source section](./structure.md#source-sections) where the [periodic columns are declared](./declarations.md). _They may only be referenced when defining integrity constraints._
+- `prod`: used to declare list folding to multiply a list of values.
 - `public_inputs`: used to declare the [source section](./structure.md#source-sections) where the [public inputs are declared](./declarations.md). _They may only be referenced when defining boundary constraints._
 - `random_values`: used to declare the [source section](./structure.md#source-sections) where the [random values are described](./declarations.md).
+- `sum`: used to declare list folding to sum a list of values.
```

Comment:
nit: I would probably adjust this to "Used to fold a list into a single value by summing all of the values in the list."